app/assets/javascripts/oxalis/controller/combinations/volumetracing_plane_controller.js

```diff
@@ -51,19 +52,23 @@ class VolumeTracingPlaneController extends PlaneControllerClass {
   componentDidMount() {
     super.componentDidMount();
 
-    let lastActiveCellId = getActiveCellId(Store.getState().tracing).get();
-    Store.subscribe(() => {
-      getActiveCellId(Store.getState().tracing).map(cellId => {
-        if (lastActiveCellId !== cellId) {
-          SceneController.renderVolumeIsosurface(cellId);
-          lastActiveCellId = cellId;
-        }
-      });
-    });
+    listenToStoreProperty(
+      state => Utils.toNullable(getActiveCellId(state.tracing)),
+      () => {
+        getActiveCellId(Store.getState().tracing).map(activeCellId =>
+          SceneController.renderVolumeIsosurface(activeCellId),
```

Comment:
Does the `renderVolumeIsosurface` code still work?

Reply:
Yeah, I also think that we would implement it differently (the current implementation was quite slow, anyway).

Reply:
I'll remove it in a separate PR :)